sendBirthdayWishes.gs

```diff
@@ -0,0 +1,93 @@
+/**
+ * Program will run on daily loop checking the reponse sheet from cANDle sign up form, to see if there are any Birthdays today.
+ * If the program finds any ANDis with their birthdays today, it will send them a personilised birthday card to their email.
+ */
+
+//This function will run on a daily loop.
+function main() {
+    // Load the sheet that contains the birthdays.
+    var ss = SpreadsheetApp.openByUrl("https://docs.google.com/spreadsheets/d/1fbgWBKx77Dy2uC7pkFw9B_MjIXDuITheYYw6vLsoC5w/edit?usp=sharing");//  Sheet Url
```

Comment:
let's try to always give meaningful names to variables :)
let's also use let and const in place of vars ;)